docs/contributing/pull-request-guidelines/index.md

```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. Select one based on the following condition.
+
+1. Standard change:
```

Comment:
Sorry, I meant to add a link here.
btw, using permanent link would be better.

```suggestion
1. [Standard change](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md):
```

Reply:
Fixed in [643252d](https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/643252d36493ee21f83eda610e31cd6de5b16ea0)